_posts/2022-09-14-Remote-dev-watch.adoc

```diff
@@ -0,0 +1,160 @@
+---
+layout: post
+title: Remote dev-watch development with WildFly Jar Maven Plugin
+date: 2022-09-14
+tags: wildfly bootable jar bootable-jar openshift cloud remote
+synopsis: Bootable JAR Maven plugin 8.0.0.Alpha2 brings in a new way to make remote development easier.
+author: ehsavoie
+---
+
+The 8.0.0.Alpha2 version of the link:https://github.com/wildfly-extras/wildfly-jar-maven-plugin/[WildFly bootable JAR Maven plugin] has been released.
+
+This https://github.com/wildfly-extras/wildfly-jar-maven-plugin/releases/tag/8.0.0.Alpha2release] is not yet Final, as it is only there to gather feedback on a new feature that simplifies development on the 'cloud' using the `dev-watch` goal.
+
+For people who are not familiar with WildFly bootable JAR and its `dev-watch` goal, I strongly recommend that you read 
+this link:https://www.wildfly.org/news/2020/12/15/bootable-jar-3.0.Beta-Released/[blog post] that covers it in details. 
+
+## Dev-watch goal
+
+The current `dev-watch` goal, although offering an efficient workflow to develop WildFly applications, requires the application or server to run locally, in the same place as the project. The improvement made on this release is to allow the application or server to run remotely so that it can be in an environment that is closer to the target runtime environment.
```

Comment:
"application or server". What do you mean? What's the application in that context? the deployment?

Reply:
bootable application or server